### ISSUE TITLE
Accessibility - WCAG 2.4.6 Headings and labels, FIELDSET element missing

### ISSUE THREAD

**Issue by bit-pro-iew-eui** [swiss/styleguide#142]
In order for screen reader users can read checkboxes and radio buttons useful, they must be grouped with the FIELDSET element (the question), it is easy to integrate.

Original version not ok
